Makefile

```
@@ -1134,3 +1139,57 @@ profiles-help:
 	@echo ""
 	@echo "See docs/FLOW_PROFILES.md for full documentation."
+
+# ============================================================================
+# Agent SDK Vendoring
+# ============================================================================
+
+.PHONY: check-agent-sdk
+check-agent-sdk:
+	@uv run --extra dev --group dev python - <<'PY'
+import sys
+try:
```

checkmake (0.2.2)

[warning] Target "try" should be declared PHONY.

(phonydeclared)